config/locales/ja.yml

```diff
@@ -486,16 +487,26 @@ ja:
     setup_webauthn_html: "\U0001F389 セキュリティ機器に対応しました！新しい機器を<a href=\"/settings/edit#security-device\">設定</a>してアカウントのセキュリティを向上しましょう。<a
       href=\"https://blog.rubygems.org/2023/08/03/level-up-using-security-devices.html\">詳細はこちら</a>！"
     api:
-      mfa_required:
-      mfa_required_not_yet_enabled:
-      mfa_required_weak_level_enabled:
-      mfa_recommended_not_yet_enabled:
-      mfa_recommended_weak_level_enabled:
+      mfa_required: このgemにはMFAの有効化が必要です。まだMFAを有効にしていません。
+      mfa_required_not_yet_enabled: |
+        ［エラー］アカウントとgemの保護のため、 https://rubygems.org/totp/new で多要素認証の設定が必要です。
+
+        詳しくはブログ記事をお読みください (https://blog.rubygems.org/2022/08/15/requiring-mfa-on-popular-gems.html)。
```

Comment:
It might be better to use `（）` instead of `()`.

Reply:
Here too.
https://github.com/rubygems/rubygems.org/pull/5272/files#diff-b85e8d609d1ca8ea9aef96cf26c7011d39b5dd2e7de34dd4c52d931979917508R498

Reply:
I think I'll use `()`. Some browsers might have a URL detection feature, and using a full width character might cause problems.